src/super_gradients/training/transforms/all_transforms.py

```diff
@@ -99,6 +101,8 @@
     Transforms.DetectionRandomRotate90: DetectionRandomRotate90,
     Transforms.DetectionHorizontalFlip: DetectionHorizontalFlip,
     Transforms.DetectionRescale: DetectionRescale,
+    Transforms.DetectionImagePermute: DetectionImagePermute,
```

Comment:
I think we have to merge Louis' PR first to use register approach. Otherwise we ran into circular import problem. This is what scary PR addressed and this is why it is so big. 
So either we merge that first and I use register, or we merge this one first but scary PR has to be updated to reflect two new transforms. @ofrimasad  @Louis-Dupont

Reply:
Ok. @Louis-Dupont  please add this to your PR